static/js/util.js

```diff
@@ -493,6 +494,37 @@ function randDomId() {
     .substr(2, 10);
 }
 
+/**
+ * add or delete statvars from url
+ *
+ * @param {string} dcid of statvar
+ * @param {boolean} add = True, delete = False
+ * @return {string} The line chart URL.
+ */
+
```

Comment:
Since this comment applies to the function, the blank line isn't necessary.

Reply:
got it.